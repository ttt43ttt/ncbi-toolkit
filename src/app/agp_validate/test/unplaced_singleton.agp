unlocalized01	1	859	1	W	chr_contig02	1	859	+
