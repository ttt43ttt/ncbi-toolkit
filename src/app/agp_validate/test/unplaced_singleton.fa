>gnl|PRJNA291306|chr_contig02 Streptomyces antibioticus strain DSM 41481 chr_contig02, whole genome shotgun sequence
AGAAGTCATCTCAATTGGCGAGTCGGCGGTAGCAGATGAGGGTGCAGGCGATGCTGGTGAAGGCGAGGAA
GTGGTCGGCCTTGCGTTCGTAGCGTCGGTGGAGTCGGCGGCAGCCGGCGAGCCAGGCCATGGTCCGTTCG
ATGGTCCAGCGGTGGCGGCCTAGTCGCTGCGAGGACTCGATGCCTCTGCGGGCTATGCGGTGCTTGATGC
CGCGCTGACGTAACCATTTCCGCAGGTGGGCGTAGTCGTATCCCTTGTCGGCGTGGAGCTTGCCGGGCCT
GCGCCGTCGGCGTCCGCGGCGTGAACGGATGGGCGGTATGCCGCGGACGAGGGGCTCGAGGGCCTGACTG
TCGTGCAGGTTCGCCCCTGAGATTCCGACGGACAGGGGCAGACCGGTCCGCTCCGTGATCAAGTGGATCT
TCGACCCGAACTTGCCCCGGTCCACAGGATTCGGACCTGTCAGGCCCCCTTTTTCAGGGCCCGCATGTTC
ACCGAGTCGATCGCGCAGCGCGACCAGTCCAGCTCACCACGGGCGCCAAGCTCATCGAGGACCAGGCGGT
GGAGCTTGGCCCACACCCGGGCCTTGCTCCACTCGGAGAACCGCCGGTGAGCTGTCGCTCCCGACGGTCC
GAACGAGGCCGTGGGCAGCTGCTGCCAGGTACAACCCGAGGTGGCGACAAACACGATCGCAGCCAGCACC
TCCCGGTCACCATGCCGACGTCGGCCACCACCCTGAGGCCGAGACGGCGCCTCCGGCACCACCCGCTGGA
ACAACTCCCACAACTCATCCGGCACCAGCCGCTCAACAATCCCCACCACGACCGACAGCTTACCCAACCG
CCCAAATGAGATGACTTCT
