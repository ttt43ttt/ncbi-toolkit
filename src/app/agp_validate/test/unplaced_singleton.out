1 component_id and length loaded from FASTA.
No runs of Ns longer than 10 bp found in FASTA sequences.
	WARNING: all objects are singletons with component_beg=1 and component_end = length of sequence

No errors, 2 warnings.
  Count Code  Description
      1  w65  all objects are singletons with component_beg=1
      -   -   no gap lines


Objects                : 1
- with single component: 1

Scaffolds              : 1
- with single component: 1

Components                   : 1 (W)
  orientation +              : 1
  orientation -              : 0
  orientation ? (formerly 0) : 0
  orientation na             : 0

Gaps                   : 0

Object names   :  1
  unlocalized01:  1

Component names:  1
  chr_contig02 :  1
1 component_id and length loaded from FASTA.
No runs of Ns longer than 10 bp found in FASTA sequences.

1:unlocalized01	1	850	1	W	chr_contig02	1	850	+
	WARNING: in unplaced singleton scaffold, component is not used in full (850 out of 859 bp)
	WARNING: all objects are singletons with component_beg=1 (not all component_end = length of sequence)

No errors, 3 warnings.
  Count Code  Description
      1  w50  in unplaced singleton scaffold, component is not used in full
      1  w65  all objects are singletons with component_beg=1
      -   -   no gap lines


Objects                : 1
- with single component: 1

Scaffolds              : 1
- with single component: 1

Components                   : 1 (W)
  orientation +              : 1
  orientation -              : 0
  orientation ? (formerly 0) : 0
  orientation na             : 0

Gaps                   : 0

Object names   :  1
  unlocalized01:  1

Component names:  1
  chr_contig02 :  1
